src/explore-education-statistics-common/src/components/form/rhf/validation/RHFCreateErrorHelper.ts

```diff
@@ -0,0 +1,70 @@
+import get from 'lodash/get';
+import {
+  FieldErrors,
+  FieldNamesMarkedBoolean,
+  FieldValues,
+} from 'react-hook-form';
+
+const RHFCreateErrorHelper = <T extends FieldValues>({
+  touched,
+  errors,
+  showErrorsForUntouchedFields = false,
+}: {
+  touched: Partial<Readonly<FieldNamesMarkedBoolean<T>>>;
+  errors: FieldErrors<T>;
+  showErrorsForUntouchedFields?: boolean;
```

Comment:
It's needed because when a Formik form is submitted it marks all the fields as touched, but RHF form doesn't do this (and has no way to simply set all as touched). To replicate the current error handling we need to know if the form has been submitted and ignore whether fields have been touched or not if it has been, so I've changed this param to `isSubmitted` to make this clearer, added a comment, and use it in getError as well as getAllErrors.

Reply:
Cool, `isSubmitted` makes a lot more sense!